src/AggregateRoot.php

```diff
@@ -6,19 +6,25 @@
 use Illuminate\Support\Str;
 use ReflectionClass;
 use ReflectionProperty;
+use Spatie\EventSourcing\Exceptions\CouldNotPersistAggregate;
 use Spatie\EventSourcing\Snapshots\Snapshot;
 use Spatie\EventSourcing\Snapshots\SnapshotRepository;
 
 abstract class AggregateRoot
 {
-    private string $aggregateUuid;
+    private string $aggregateUuid = '';
```

Comment:
Rename aggregateProperties without "aggregate", since we are in an aggregate class

Reply:
Rename `aggregateUuid` to `uuid`.